docs/sql_reference/src/asciidoc/_chapters/sql_functions_and_expressions.adoc

```diff
@@ -8829,6 +8829,222 @@ with the character set specified by _char-set-name_.
 SPACE(3)
 ```
 
+<<<
+[[split_part_function]]
+== SPLIT_PART Function
+
+The SPLIT_PART function returns the n^th^ substring that is split on the defined `_delimiter_` and `_filed-position_`.
+
+SPLIT_PART is a Trafodion SQL extension.
+
+`SPLIT_PART (_source-string, delimiter, filed-position_)`
+
+[[syntax_description_of_split_part]]
+=== Syntax Description of SPLIT_PART
+
+* `_source-string_`
++
+is a string to be split. 
++
+NOTE: The `_source-string_` must be character and cannot be NULL.
+
+* `_delimiter_`
++
+is a delimiter to split a `_source-string_`.
++
+The delimiter can contain multiple characters, not just a single character. Multiple characters are considered as a whole, not individually. For example, if the delimiter is "coder", then the source-string is split around "coder" rather than "c", "o", "d", "e" and "r".
```

Comment:
Thanks for this note! It's a subtle point.

Reply:
:raising_hand: 💯 😁